postcss.config.js

```diff
@@ -1,3 +0,0 @@
-const config = require('./packages/terra-clinical-site/postcss.config');
```

Comment:
Is this meant to be removed in this PR? Seems like the scope of it is for removing props-table scripts.

Reply:
Yeah I am moving this to a seperate pr